BuberBreakfast/Pages/Shop/View.cshtml

```diff
@@ -0,0 +1,44 @@
+@page
+@model BuberBreakfast.Pages.Shops.ShopPageModel
+<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-QWTKZyjpPEjISv5WaRU9OFeRpok6YctnYmDr5pNlyT2bRjXh0JMhjY6hW+ALEwIH" crossorigin="anonymous">
+<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js" integrity="sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz" crossorigin="anonymous"></script>
+
+<p>Number of shops: @Model.Shops.Count</p>
```

Comment:
# Important
`@Model` allows us to access all the public variables in `BuberBreakfast.Pages.Shops.ShopPageModel`